.github/workflows/pre-commit.yml

```diff
@@ -23,6 +23,12 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: "3.9"
+      - name: Format python code with black
+        uses: psf/black@stable
+        with:
+          options: "--check --verbose"
+          src: "./scripts"
+          version: "~= 22.0"
```

Comment:
Hi @za ,
Can you remove this and update the precommit config instead please? This is  the GitHub workflow.

Reply:
See https://github.com/OWASP/wrongsecrets/blob/master/.pre-commit-config.yaml

Reply:
> Hi @za , Can you remove this and update the precommit config instead please? This is the GitHub workflow.

Removed.